src/components/breadcrumb/components/item/item.stories.js

```diff
@@ -0,0 +1,12 @@
+import { html } from 'lit-html';
+import '../..';
+import styles from '../../styles.scss';
+import { stylesheet } from '../../../../../.storybook/utils';
+
+export default {
+  title: 'native-elements/vl-breadcrumb/vl-breadcrumb-item',
```

Comment:
De `vl-breadcrumb` staat nu nog onder de `native-elements` maar mag onder de `custom elements`, dit stond nog zo omdat de component die we refactored hebben een native element was.

Reply:
Hiermee ga je ook de get in je testen moeten wijzigen naar custom-elements iso native-elements.